src/hmem.c

```diff
@@ -170,3 +175,35 @@ void ofi_hmem_cleanup(void)
 			hmem_ops[iface].cleanup();
 	}
 }
+
+enum fi_hmem_iface ofi_get_hmem_iface(const void *addr)
```

Comment:
Have you come across any situation where we would need this outside of providers that want to handle GPU buffers without requiring FI_MR_HMEM?